response_operations_ui/controllers/survey_controllers.py

```diff
@@ -0,0 +1,24 @@
+import json
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_surveys_list():
+    logger.debug('Retrieving surveys list')
+    url = '{}/{}'.format(app.config['BACKSTAGE_API_URL'], 'survey/surveys')
+
+    response = requests.get(url)
+    if response.status_code != 200:
+        logger.error('Failed to retrieve surveys list')
```

Comment:
Is there value in logging here as `@app.errorhandler(ApiError)` will also log?

Reply:
probably not, I'll remove